gcp/live/dev/k8s/kube-system/k8s-snapshots/terraform.tfvars

```diff
@@ -0,0 +1,16 @@
+# ↓ Module metadata
+terragrunt = {
+  terraform {
+    source = "/project/modules//k8s-snapshots"
+  }
+
+  dependencies {
```

Comment:
This dependency is not needed.

Reply:
It was needed, because we wanted to patch volumes after they are created. Not anymore.